plugin/src/main/java/org/opensearch/ml/searchext/MLInferenceRequestParametersExtBuilder.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.ml.searchext;
+
+import static org.opensearch.ml.searchext.MLInferenceRequestParameters.ML_INFERENCE_FIELD;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.xcontent.XContentBuilder;
+import org.opensearch.core.xcontent.XContentParser;
+import org.opensearch.search.SearchExtBuilder;
+
+public class MLInferenceRequestParametersExtBuilder extends SearchExtBuilder {
+    private static final Logger logger = LogManager.getLogger(MLInferenceRequestParametersExtBuilder.class);
+    public static final String NAME = ML_INFERENCE_FIELD;
```

Comment:
you can use ML_INFERENCE_FIELD directly, NAME is not needed

Reply:
I was following other search extension format, all have a .NAME so trying to make it follow the same format